bedrock/firefox/templates/firefox/sync.html

```diff
@@ -0,0 +1,81 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% from "macros-protocol.html" import hero, picto_card, feature_card with context %}
+
+{% block page_title %}{{ ftl('firefox-sync-firefox-sync') }}{% endblock %}
+
+{% block page_desc %}{{ ftl('firefox-sync-access-and-sync') }}{% endblock %}
+
+{% block page_css %}
+ {{ css_bundle('sync') }}
+{% endblock %}
+
+{% block content %}
+
+{% call hero(
+  title=ftl('firefox-sync-sync-your-firefox'),
+  class='mzp-has-image mzp-t-firefox mzp-t-product-firefox',
+  image_url='img/firefox/accounts/trailhead/value-knowledge.jpg',
+  include_highres_image=True,
+  include_cta=True,
+  heading_level=1
+  ) %}
+  <p>{{ ftl('firefox-sync-access-and-sync') }}</p>
+
+  {{ download_firefox_thanks() }}
+  <p class="fxa-cta">
+    {{ fxa_button(
```

Comment:
I think you still need to include the JS dependencies for the FxA button? https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#cta-button-dependencies